### PR TITLE
Remove messager_app compose prd + minimal stability

### PR DESCRIPTION
**Pull request by melroy89**
- Remove `messenger_app` from `compose.prd.yml`
- Set minimum stability to stable in `composer.json` + fix stability
- Add `tools` directory to Docker ignore
- Add missing `"cache:clear": "symfony-cmd"` back to `composer.json` (this is how a new Symfony app suppose to look like)

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -163,15 +163,16 @@
         }
     },
     "scripts": {
+        "auto-scripts": {
+            "cache:clear": "symfony-cmd",
+            "assets:install %PUBLIC_DIR%": "symfony-cmd"
+        },
```
Comment (BentiGorlich) [inline composer.json:169]: I don't know what exactly this is doing

Comment (BentiGorlich) [reply composer.json:169]: ah ok :+1:

**Review by BentiGorlich — COMMENTED** [review]
I am going to just run a quick test of the new composer packages tomorrow. I think it will just be fine, but I wanna be sure :grin:

**Comment by melroy89** [comment]
> I am going to just run a quick test of the new composer packages tomorrow. I think it will just be fine, but I wanna be sure 😁

Thanks for testing thoroughly

**Review by BentiGorlich — APPROVED** [review]
LGTM. The new dependencies made no obvious problem, so maybe its in the details, but I don't think so :D

**Comment by melroy89** [comment]
> LGTM. The new dependencies made no obvious problem, so maybe its in the details, but I don't think so :D

Yea I just applying symfony & composer best practices.

**Pull request merged**